### ISSUE TITLE
Editor option for Text boxes

### ISSUE THREAD

**Issue by javulorg** [javulorg/javul#62]
Task information>Other links	

Original comment by Shiva:
The "Editor" should contain underline(U) option.	

Additional comment by Ed: 
Ideally I would like editor for all other contents on the site (Unit, Objective, Task, Issue) and Wiki, Forum, Inbox messages to use the same editor. Investigate options to use the best editor that also includes Underline?
So this needs some work on looking into available open-source editors and integrating them with all Edit boxes on the site.

Screenshot by Shiva for missing Underline feature;
https://www.screencast.com/t/UdjOyIzY

**Comment by javulorg** [comment]
Igor
I think the first step here would be to look at available open-source text box editors. I like the visual formatting of the default editor (e.g. which is used to edit a Unit page). Maybe it has additional options?
Ed

**Comment by Motordread** [comment]
I'm not closing this issue, will need more investigation about WYSIWYG editor that we can customize.

**Comment by javulorg** [comment]
Here's a Google search result for "laravel WYSIWYG  editor"
https://www.google.com/search?q=laravel+text+editor&oq=laravel+text+editor&aqs=chrome..69i57j0l5.2312j0j7&sourceid=chrome&ie=UTF-8#safe=off&q=laravel+WYSIWYG++editor

I will ask the first web team what they used. Maybe that editor has additional options too.

**Comment by javulorg** [comment]
Yes the first web team confirmed that Summernote is the one they used.
I would like these added: Style, Paragraph, Table, Image
'Image' might require setup for uploading images.

**Comment by javulorg** [comment]
hi Igor
This is very good. Thank you. I see though that the image data is being embedded into the webpage itself instead of being uploaded as a file. Test page: http://javul.org/wiki/home/qdQJNqgr83/test
Source code:  ![image](https://user-images.githubusercontent.com/19337355/27716522-f51cc74c-5d0c-11e7-92f0-8b3fb489cbd1.png)

Thats interesting as its the first time I've seen image data embedded in the webpage itself. I did a Google search for "summernote image upload" and I found this link where they talk about uploading to a folder (this is how I would like):
https://stackoverflow.com/questions/21628222/summernote-image-upload

This should be good enough for now although in the future we can try to have additional options (auto resize, upload to hashed subdirectory to manage large amount of files).
